api/tests/opentrons/protocol_runner/smoke_tests/test_protocol_runner.py

```diff
@@ -28,6 +30,9 @@
 from opentrons.protocol_runner import create_simulating_runner
 
 
+# TODO (tz, 6-17-22): API version 3.x in-development.
+# Currently parsing protocol versions less then MAX_SUPPORTED_VERSION
+@pytest.mark.xfail
```

Comment:
```suggestion
@pytest.mark.xfail(strict=True)
```

So that if this starts to work, this test will raise an "unexpected pass" message and prompt us to remove this `# TODO` comment.